kvx/meson.build

```diff
@@ -0,0 +1,20 @@
+project('kvx', 'c')
+
+rz_asm_dep = dependency('rz_asm')
+rz_analysis_dep = dependency('rz_asm')
```

Comment:
Probably you want to use `rz_analysis_dep = dependency('rz_analysis')` ?

Reply:
yes, right.